versioned_docs/version-1.2.0/install-kubeslice/yaml/slice-operations/slice-operations-slice-creation.mdx

```diff
@@ -0,0 +1,916 @@
+renewal procedure before the rotation interval expires. The renewBefore parameter in the YAML file can be set to a suitable value to 
+ensure that the certificate renewal process begins right away, maintaining your system's security and compliance.
+
+
+:::warning
+Before you use the renewBefore parameter to renew the gateway certificates, ensure the slice gateways are operational.
+:::
+
+Use the following template to renew your certificates.
+
+```
+apiVersion: controller.kubeslice.io/v1alpha1
+kind: SliceConfig
+metadata: 
+  name: <slice-name>
+  namespace: kubeslice-<projectname>
+spec:
+  sliceSubnet: <slice-subnet>
+  maxClusters: <2 - 32>       #Ex: 5. By default, the maxClusters value is set to 16
+  sliceType: Application
+  sliceGatewayProvider:
+    sliceGatewayType: OpenVPN
+    sliceCaType: Local
+  sliceIpamType: Local
+  renewBefore: 2023-07-18T14:27:08Z        #only required if you want to renew your certificates before hand 
+  rotationInterval: 60    # If not provided, by default key rotation interval is 30 days
+  clusters:
+    - <registered-cluster-name-1>
+    - <registered-cluster-name-2>
+  qosProfileDetails:
+    queueType: HTB
+    priority: <qos_priority>                      #keep integer values from 0 to 3
+    tcType: BANDWIDTH_CONTROL
+    bandwidthCeilingKbps: 5120
+    bandwidthGuaranteedKbps: 2560
+    dscpClass: AF11
+  namespaceIsolationProfile:
+    applicationNamespaces:
+     - namespace: iperf
+       clusters:
+       - '*'
+    isolationEnabled: false                   #make this true in case you want to enable isolation
+    allowedNamespaces:
+     - namespace: kube-system
+       clusters:
+       - '*'
+  ```
+
+## Inter-Cluster Connectivity in KubeSlice
+KubeSlice supports cluster connectivity among public and private clusters. Kubeslice extends the support to private clusters 
+using a Load Balancer. In the slice configuration YAML file, the user can specify the type of gateway connectivity to use 
+on a cluster. The type can be either NodePort or LoadBalancer. If you do not use LoadBalancer as the gateway connectivity 
+type, then NodePort is used by default. Like other configuration options in KubeSlice, the * (asterisk) wildcard 
+character is allowed to indicate all clusters of a slice.
+
+Add the gateway service type and protocol and the corresponding clusters under the sliceGatewayServiceType in the slice 
+configuration YAML as illustrated below.
+
+ ```
+ sliceGatewayProvider:
+    sliceGatewayType: OpenVPN
+    sliceGatewayServiceType:
+      - cluster: <registered-cluster-name-1>
+        type: LoadBalancer   # By default, the gateway service type is NodePort.
+        protocol: TCP        # By default. the gateway protocol is UDP.
+      - cluster: <registered-cluster-name-2>
+        type: LoadBalancer   # By default, the gateway service type is NodePort.
+        protocol: TCP        # By default, the gateway protocol is UDP.   
+```
```

Comment:
This was added as a sub-heading in EE v1.5.0: https://docs.avesha.io/documentation/enterprise/1.5.0/install-kubeslice/yaml/slice-operations/slice-operations-slice-creation#change-the-gateway-provider-for-inter-cluster-connectivity. 

They are in the EE v1.5.0 source docs. Can you copy that content here?

Reply:
done